src/lib.rs

```diff
@@ -330,23 +330,8 @@ where
     F: Fn() + Send + Sync + 'static,
```

Comment:
Remove the comment on lines 326-327.  (The comment should go in the `new` methods for the Random and PCT schedulers.)